spec/requests/api/merge_requests_spec.rb

```diff
@@ -342,6 +357,12 @@
       json_response['note'].should == 'My comment'
     end
 
+    it "should return comment for merge_request by iid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.